yarn.lock

```diff
@@ -2,381 +2,370 @@
 # yarn lockfile v1
```

Comment:
Since there are no updates to the dependencies in package.json, you can omit this yarn.lock from this PR. Yarn.lock locks in the versions of the installed packages shown on package.json. That way when someone else installs the project, they have the same exact dependency libraries as everyone else and they won't have any unexpected errors from version mismatch.

We normally include this yarn.lock file if the feature branch has an update on one of the dependences and we want everyone else to update to the same stable version. Like maybe you needed a new feature in chartjs that is avail on the latest version so you need to update chartjs from 1.3 to 1.7 or something.

Reply:
Thanks for explaining that. Sorry if it's a dumb question but what is the best way to correct this? Meaning like just remove yarn.lock from the commit? Or is there another specific way to do this?

Reply:
So, I'm guessing this might due to my inexperience but when I just deleted yarn.lock the build failed. I cloned the yarn.lock from master and added it back in and everything is working. Please just let me know what I should actually be doing. Thanks!

Reply:
That's a great question actually! Lol I had no idea what to do starting out as well.

Since you already saved it to your commit history, I believe the best way is what you just did, clone it from the master. Yarn.lock is crucial to your project. If it is missing, you can do a yarn install and it will be replaced, but with possibly an updated version from github. If you clone it from master and then do a yarn install, yarn will then retrieve all the needed dependencies with the exact same version described on the yarn.lock.

Reply:
Thanks again, I didn't think about doing a checkout on just that file as mentioned above and I will definitely try that going forward!